content/applications/sales/subscriptions.rst

```diff
@@ -4,20 +4,22 @@
 Subscriptions
 =============
 
-**Odoo Subscriptions** is used to run recurring businesses: :ref:`sell new contracts
-<subscriptions/quotations>`, :doc:`upsell customers <subscriptions/upselling>`, keep the churn under
-control, and :doc:`generate reports <subscriptions/reports>` on the main KPIs: MRR, ARR, retention,
-churn, upselling, etc.
+Odoo **Subscriptions** is used to run recurring businesses: :ref:`sell new contracts
+<subscriptions/quotations>`, :doc:`upsell customers </applications/sales/subscriptions/upselling>`,
+keep the churn under control, and :doc:`generate reports
+</applications/sales/subscriptions/reports>` on the main KPIs: MRR, ARR, retention, churn,
+upselling, etc.
 
 .. seealso::
    - `Odoo Tutorials: Subscriptions <https://www.odoo.com/slides/subscription-20>`_
-   - :doc:`subscriptions/products`
-   - :doc:`subscriptions/plans`
-   - :doc:`subscriptions/upselling`
-   - :doc:`subscriptions/renewals`
-   - :doc:`subscriptions/closing`
-   - :doc:`subscriptions/automatic_alerts`
-   - :doc:`subscriptions/reports`
+   - :doc:`/applications/sales/subscriptions/products`
+   - :doc:`/applications/sales/subscriptions/ecommerce`
+   - :doc:`/applications/sales/subscriptions/plans`
+   - :doc:`/applications/sales/subscriptions/upselling`
+   - :doc:`/applications/sales/subscriptions/renewals`
+   - :doc:`/applications/sales/subscriptions/closing`
+   - :doc:`/applications/sales/subscriptions/automatic_alerts`
+   - :doc:`/applications/sales/subscriptions/reports`
```

Comment:
Not sure if this was intentional, since **toctree** links are at the bottom, but I think I like the redundancy actually 👍